packages/code-studio/src/log/LogExport.ts

```diff
@@ -6,6 +6,42 @@ import { logHistory } from './LogInit';
 
 const FILENAME_DATE_FORMAT = 'yyyy-MM-dd-HHmmss';
 
+// List of objects to blacklist
+// '' represents the root object
+export const PATH_BLACKLIST: string[][] = [
+  ['api'],
+  ['client'],
+  ['dashboardData', 'defaultLayout', 'connection'],
+  ['layoutStorage'],
+  ['storage'],
+].map(path => ['', ...path]);
```

Comment:
This isn't good either - you'd need to do the same thing when passing in your own blacklist. If I'm using the `exportLogs` function below, calling `exportLogs('myFilePrefix', {}, ['client'])`, I'd expect the path `client` to not appear in the exported logs. Requiring the consumer of the API to pass in `['', 'client']` to achieve that is a [footgun](https://en.wiktionary.org/wiki/footgun).

I'd remove the `''` string from the current path when you compare it to the blacklist. Why is there an empty string at the front anyway?

Reply:
The empty string represents the key of the initial object